README.md

```diff
@@ -91,19 +103,74 @@ See [example docker-compose.yml](docker-compose.yml).
 
 ## Run RCON commands
 
-Open a shell into your container via `docker exec -ti palworld-dedicated-server bash`, then you can run commands against the gameserver via the command `rcon` or `rconcli`
+Open a shell into your container via `docker exec -ti palworld-dedicated-server bash`, then you can run commands against the gameserver via the command `rconcli`
 
 ```shell
-$:~/steamcmd$ rcon showplayers
+$:~/steamcmd$ rconcli showplayers
 name,playeruid,steamid
-$:~/steamcmd$ rcon info
+$:~/steamcmd$ rconcli info
 Welcome to Pal Server[v0.1.3.0] jammsen-docker-generated-20384
-$:~/steamcmd$ rcon save
+$:~/steamcmd$ rconcli save
+Complete Save
+```
+
+You can also use `docker exec palworld-dedicated-server rconcli <command>` right on your terminal/shell.
+
+```shell
+$ docker exec palworld-dedicated-server rconcli showplayers
+name,playeruid,steamid
+
+$ docker exec palworld-dedicated-server rconcli info
+Welcome to Pal Server[v0.1.3.0] jammsen-docker-generated-20384
+
+$ docker exec palworld-dedicated-server rconcli save
 Complete Save
 ```
 
 > **Important:** Please research the RCON-Commands on the official source: https://tech.palworldgame.com/server-commands
 
+## Backup Manager
+
+Usage: `docker exec palworld-dedicated-server backup [command] [arguments]`
```

Comment:
Move usage to example down below and the WARNING up here.
Better flow for readability